src/engine/game/gui/ingame/equip.h

```diff
@@ -18,6 +18,12 @@
 #pragma once
 
 #include "../gui.h"
+#include "../../../gui/control/button.h"
+#include "../../../gui/control/imagebutton.h"
+#include "../../../gui/control/label.h"
+#include "../../../gui/control/listbox.h"
+
+using namespace reone::gui;
```

Comment:
`using namespace` in header files pollutes the namespace and potentially breaks things. Needs to be `gui::Label` and the like below.